### PR TITLE
Javascript/javascript3/week1

### PR DESCRIPTION
**Pull request by Bhuvaneshwari29**
JS3 week1 homework completed

### CODE REVIEW

File: javascript/javascript3/week1/scriptgif.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        const img = document.createElement("img");
+        console.log(element.images.downsized_still.url);
+        img.width = "200";
+        img.height = "200";
+        img.src = element.images.downsized_still.url;
+        outputDivTag.appendChild(lineBreak);
+        outputDivTag.appendChild(img);
+      });
+    })
+    .catch((err) => {
```
Comment (arya56) [inline javascript/javascript3/week1/scriptgif.js:22]: Nice using catch.

**Review by arya56 — COMMENTED** [review]
@Bhuvaneshwari29  Great job :+1:  very clean & tidy result in appearance.(HTML & css) Just few minor hints has been suggested. Based on homework source, wind speed could be added.

File: javascript/javascript3/week1/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  detailsBox.style.visibility = "visible";
+  fetch(
+    `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=metric&appid=f289a9857cc75d109e0ec14d9b852abb`
+  )
+    .then((response) => response.json())
+    .then((weatherData) => {
+      console.log(weatherData);
+      renderWeatherStatus(weatherData);
+    });
+}
```
[the remark this answers is not in the document]
Comment (Bhuvaneshwari29) [reply javascript/javascript3/week1/index.js:35]: Yes, That's a valid question. Will make the changes to add the functionality to check if the city exist or not and throw error.

File: javascript/javascript3/week1/index.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  }
+  if (temperature >= 20) {
+    message.innerText = `No Jackets. Enjoy the weather`;
+  }
+  let iconCode = weatherData.weather[0].icon;
+  console.log(iconCode);
+  let iconurl = "http://openweathermap.org/img/w/" + iconCode + ".png";
+  icon.src = iconurl;
+
+  let cloudiness = weatherData.clouds.all;
```
[the remark this answers is not in the document]
Comment (Bhuvaneshwari29) [reply javascript/javascript3/week1/index.js:58]: Changed to const. Thanks for the link to the article.

File: javascript/javascript3/week1/index.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+weatherInput.addEventListener("click", getCityWeather);
+function getCityWeather() {
+  let cityName = city.value;
+  detailsBox.style.visibility = "visible";
+  fetch(
+    `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=metric&appid=f289a9857cc75d109e0ec14d9b852abb`
+  )
+    .then((response) => response.json())
+    .then((weatherData) => {
+      console.log(weatherData);
```
[the remark this answers is not in the document]
Comment (Bhuvaneshwari29) [reply javascript/javascript3/week1/index.js:32]: Yes, Done .

File: javascript/javascript3/week1/index.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    hour: "2-digit",
+    minute: "2-digit",
+  });
+  sunrise.innerText = actualSunriseTime;
+
+  let sunsetTime = weatherData.sys.sunset;
+  console.log(sunsetTime);
+  let utcMilliSecSunset = sunsetTime * 1000;
+  let actualSunsetTime = new Date(utcMilliSecSunset);
+  actualSunsetTime = actualSunsetTime.toLocaleTimeString([], {
```
[the remark this answers is not in the document]
Comment (Bhuvaneshwari29) [reply javascript/javascript3/week1/index.js:78]: Thank you :-)

File: javascript/javascript3/week1/scriptgif.js (hunk 1 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript3/week1/scriptgif.js:22]: Thank you :-)

**Pull request merged**